comfy-mobile-ui-api-extension/utils/global_websocket_manager.py

```
@@ -215,8 +333,10 @@ async def add_client(self, ws: web.WebSocketResponse):
                     for event, data in self.last_events.items():
                         if event == 'execution_error':
                             continue
-                        message = {"type": event, "data": data}
-                        await ws.send_str(json.dumps(message))
+                        json_msg = self._build_json_message(event, data)
+                        if json_msg is None:
+                            continue
+                        await ws.send_str(json_msg)
                 except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)